<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>897</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_set_color_threshold">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>800</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(112, 133, 255);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(110, 33, 255);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Set Color Threshold</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse_img">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>90</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(112, 133, 255);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(110, 33, 255);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_generate_mask">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>800</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(112, 133, 255);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(110, 33, 255);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Generate Mask</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="image_path_textedit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>631</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Image path:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_image_display">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>631</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse_img_folder">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>170</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(112, 133, 255);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(110, 33, 255);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="image_folder_path_textedit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>631</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Image Folder:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_first_img">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>680</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_last_img">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>680</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_prev_img">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>680</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next_img">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>680</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cnt">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>690</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_edit_masks">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>800</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(112, 133, 255);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(110, 33, 255);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Edit Masks</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
